src/DI.php

```diff
@@ -219,6 +220,11 @@ public static function session()
 		return self::$dice->create(Core\Session\Capability\IHandleSessions::class);
 	}
 
+	public static function sessionUser(): IHandleUserSessions
```

Comment:
Cannot we rename this to `userSession` to reflect the same renaming for class `UserSession`?